### PR TITLE
README update

### PR DESCRIPTION
**Pull request by TamiTakamiya**
1. Replace 
>  the `ftdata.json` file, which is the training dataset for developing custom AI models used with Ansible Lightspeed.

with

> the `ftdata.json` file, which is the training dataset for developing custom AI models.

following the suggestion from Marty.

2. Insert the installation steps from User Guide draft. Because this README will be viewable on GitHub and PyPI, readers are mostly engineers and they should be familiar with how pip works and these steps look too redundant.  **However, I still want to insert them to README here** because it would be pretty likely many of them already have Ansible libraries on their system and I would like them to install `ansible-content-parser` in an isolated virtual environment.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,8 +16,73 @@ the `dist` directory.
 
 ## Installation
 
+### Prerequisites
+
+- Python version 3.10 or later.
+- UNIX OS, such as Linux or Mac OS.
+  ` **Note:** Installation on Microsoft Windows OS is not supported.
```
Comment (robinbobbitt) [inline README.md:23]: stray backtick here

File: README.md [hunk 2]
```diff
@@ -16,8 +16,73 @@ the `dist` directory.
...
+
+`ansible-content-parser` uses a newer version of `ansible-lint` and its
+dependent components. In order to isolate them from the existing
+Ansible installations, it is recommended to install `ansible-content-parser` in
+a Python virtual environment with the following steps:
+
+1. Create a working directory and set up venv Python virtual environment:
+
+```commandline
+$ python -m venv ./venv
```
Comment (robinbobbitt) [inline README.md:35]: Shall we take out the $s? They make a copy-paste unusable.

**Review by robinbobbitt — CHANGES_REQUESTED** [review]
These are fantastic directions and worked well for me! Just a couple minor adjustments suggested.

File: README.md (hunk 1 above)
Comment (TamiTakamiya) [reply README.md:23]: Thanks. Fixed it.

File: README.md (hunk 2 above)
Comment (TamiTakamiya) [reply README.md]: Removed '$'s.  I kept '$'s in the execution examples (outputs shown with the `--version` and `--help` options.)

**Comment by TamiTakamiya** [comment]
@robinbobbitt Thanks for your comments. I have addressed your comments in the latest commit. Would you take a look? Thank you.

**Review by robinbobbitt — APPROVED** [review]
Thanks @TamiTakamiya !

**Pull request merged**